docs/install.md

```diff
@@ -0,0 +1,132 @@
+#Installation and Settings
+The MongoDB module and sub-modules need some configuration to be useful. This
+guide assumes that :
+
+* a [MongoDB][download] 3.0 to 3.6 instance is already installed, configured, and
+  available to connect to from the Drupal instance.
+* the site will be running [Drupal][drupal] 8.[56].x, with [Drush][drush] 8.x.
+* the [mongodb][mongodb] (not [mongo][mongo]) PHP extension version 1.1.7 or
+  later is installed and configured.
+* PHP is version 7.[01].x. At this point, [PHP 7.2.x][php72] might not pass the
+  test suite. It should be compatible by the time Drupal 8.6.0 is released.
+
+[download]: https://www.mongodb.org/downloads
+[drupal]: https://www.drupal.org/project/drupal
+[drush]: https://www.drupal.org/project/drush
+[php]: http://php.net/downloads.php
+[mongo]: http://php.net/mongo
+[mongodb]: http://php.net/mongodb
+[php72]: https://www.drupal.org/node/2936045
+
+If MongoDB 3.0 to 3.5 is installed on `localhost:27017` and `mongod` was started
+with the `–httpinterface` option, you may view the web admin interface:
+
+    http://localhost:28017/
+
+* Download the module package, as per
```

Comment:
This whole section is a bit too long: it should be split in subsections with subheadings to make it easier to read: we could have parts like "prerequisites" or "requirements", then "code deployment", then "settings configuration", etc.